CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Cover `src/components/Loading/Loading.js` file by RTL/Jest tests. Refs UIREQ-885.
 * Leverage cookie-based authentication in all API requests. Refs UIREQ-861.
 * Update `circulation` okapi interface to `14.0` version. Refs UIREQ-954.
+* Added requestDate token. Refs CIRC-1784.
```

Comment:
Hello how i understood CIRC (mod-circulation) this is BE Story, you should have corresponding FE Story and provide your changes in scope of FE Story in current case UIREQ (ui-requests)

Reply:
I do not have separate FE story. Also, the FE change is spread across three different repositories. ui-circulation, ui-request and ui-checkin.
I am just adding a new token for requestDate.